INFO     cardano_node_tests.tests.conftest:conftest.py:135 Changed CWD to '/home/martink/Source/repos/cardano-node-tests5/tmp/pytest-of-martink/pytest-32/popen-gw2'.
INFO     cardano_node_tests.utils.cluster_management:cluster_management.py:443 Stopping cluster with `/home/martink/Source/repos/cardano-node-tests5/tmp/pytest-of-martink/pytest-32/cluster8/startup_files/nsimdxuj/stop-cluster-hfc`.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:44 Start script coverage log file saved to '/home/martink/Source/repos/cardano-node-tests5/.cli_coverage/cli_coverage_script_220725_194234633_cfyf.log'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:80 Cluster artifacts saved to '/home/martink/Source/repos/cardano-node-tests5/tmp/pytest-of-martink/pytest-32/cluster_artifacts/state-cluster8_fnywnjtu'.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:344 Starting cluster with `/home/martink/Source/repos/cardano-node-tests5/tmp/pytest-of-martink/pytest-32/cluster8/startup_files/nsimdxuj/start-cluster-hfc`.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:346 Cluster started.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:368 Performing 'restart' action on services ['nodes:pool2', 'nodes:pool3'].
INFO     cardano_node_tests.tests.test_kes:test_kes.py:245 2022-07-25 19:59:47.633628: Waiting for 200 sec for KES expiration.
INFO     cardano_node_tests.tests.test_kes:test_kes.py:249 2022-07-25 20:03:07.733805: KES expired (?); tip: '{'block': 559, 'epoch': 6, 'era': 'Alonzo', 'hash': 'c22f7f0cc06de8cd3bc2a29274e4cb3b80083bd8f9c46109db07b3f1d1840996', 'slot': 5239, 'syncProgress': '100.00'}'.
INFO     cardano_node_tests.tests.test_kes:test_kes.py:130 2022-07-25 20:03:07.771076: Waiting for 1 new epoch(s).
INFO     cardano_clusterlib.clusterlib_klass:clusterlib_klass.py:3099 Waiting for 173.20 sec for slot no 6105.
INFO     cardano_node_tests.tests.test_kes:test_kes.py:133 2022-07-25 20:06:04.947091: Waiting for the end of current epoch.
INFO     cardano_node_tests.utils.cluster_nodes:cluster_nodes.py:368 Performing 'restart' action on services ['nodes:pool2', 'nodes:pool3'].
INFO     cardano_node_tests.tests.test_kes:test_kes.py:267 2022-07-25 20:09:07.633289: Waiting 120 secs to make sure the expected errors make it to log files.